validator/sawtooth_validator/metrics/metrics.py

```diff
@@ -0,0 +1,196 @@
+# Copyright 2018 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ------------------------------------------------------------------------------
+
+import os
+import platform
+
+
+class MetricsConfig:
+    DEBUG = 10
```

Comment:
Are these reporting levels?  Should they be "constant" instances of `MetricsConfig`?  e.g. `DEBUG = MetricsConfig(10)`

Reply:
This config will be extended in a future PR to include per-module configuration, so it doesn't make sense to have constant instances. You also need to be able to specify a reporting level when you create a metric.